### ISSUE TITLE
Git ignore .idea directory

### ISSUE THREAD

**Issue by steelbrain** [mattkrick/meatier#133]
`.idea` is a user-specific configuration directory by IntelliJ IDEs and should not be included in git repos

**Comment by mattkrick** [comment]
duplicate of https://github.com/mattkrick/meatier/pull/19

**Comment by steelbrain** [comment]
Thanks for pointing it out but I searched for open/closed *issues* before opening and couldn't find any :)

Are you open to a discussion about this?

Because in my opinion, including `.idea` directory in your project is not the best idea ever. IDE plugins and even the IDE itself in some cases, store secret tokens or private data in `.idea` they'll be published when you do `git add .`

**Comment by mattkrick** [comment]
sure, i'm always open for discussion 😄 
there's no private data in there, just a lotta xml on custom rules that are set up (eg ignore certain things that it thinks are HTML errors but are totally legit in JSX, or flag `universal` as a resource root, etc). 
In my mind, it's exactly the same as a dotfile & saying it doesn't belong is like saying you shouldn't include your `.github`. Will you ever use the `.github` folder when you clone the repo? Heck no! But there's actually a chance you'll use this, so it's actually more useful.

Additionally, when someone troubleshoots a problem & they say "screw it, i'll do a fresh clone" they shouldn't have to blow an hour to re-setup their entire webstorm project config. 

TL;DR: you're gonna delete the `.github` folder anyways. Just delete this one, too.

**Comment by steelbrain** [comment]
Please have a look at [`the official doc`](http://www.jetbrains.com/help/idea/2016.1/project.html?origin=old_help) discussing this
